scrypt.1

```diff
@@ -83,8 +83,9 @@ does not print any prompts, and reads a passphrase from stdin.
 .Sh OPTIONS
 .Bl -tag -width "-m maxmemfrac"
 .It Fl f
-Force the decryption to proceed even if it is anticipated to
-require an excessive amount of memory or CPU time.
+Force the operation to proceed even if it is anticipated to
+require an excessive amount of memory or CPU time.  Do not print
+any warnings about exceeding any memory or CPU time limits.
```

Comment:
This arose from
https://github.com/Tarsnap/scrypt/issues/316#issuecomment-908175816
but the change was sitting around in a spare branch for a while.